### PR TITLE
lint

### PR DESCRIPTION
**Pull request by MicheleBertoli**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "airbnb",
+  "env": {
+    "mocha": true
+  },
+  "rules": {
+    "semi": [2, "never"],
+    "no-param-reassign": 0
```
Comment (cef62) [inline .eslintrc]: I'd like to have this rule enabled, there's some particular reason to disable it?

**Comment by cef62** [comment]
Great Michele! I've just two rules I'd like to discuss:
- `quotes` I don't like too much this rule because it prevents also using template string without insertion inside
- `no-param-reassign` I'd prefer this enabled

File: .eslintrc (hunk 1 above)
Comment (MicheleBertoli) [inline .eslintrc]: We are mutating [traversedPath.node.leadingComments](https://github.com/cef62/css-modules-compiler/blob/master/src/update-css-modules-imports.js#L46) and I usually avoid polluting the code with eslint-related comments.
Do you prefer me to disable the rule for that line only?

**Comment by MicheleBertoli** [comment]
Thanks @cef62!

Any reason why we should be using template literals without variables?
In case, what about `"quotes": [2, "single", {"allowTemplateLiterals": true}]`?

**Comment by cef62** [comment]
The last rule you proposed it's ok for me 👍 

The rationale behind my request is that often happens you write a plain string using single quotes and later you need to add dynamic information to it and therefore you must switch from single quotes to backtics. To avoid this I use backticks most of the times.

**Comment by MicheleBertoli** [comment]
Got it, I usually tend to avoid solving future problems and I like to make explicit changes (i.e. single to backtick) but I see your point.
I will update the PR with the new rule.

What about [this](https://github.com/cef62/css-modules-compiler/pull/6#discussion_r69858704)? Should I use `eslint-disable-next-line`?

Thanks!

File: .eslintrc (hunk 1 above)
Comment (cef62) [inline .eslintrc]: You're right, I also don't like polluting code with eslint related comments but _maybe_ in this case is better. This mutation is the official way to change comments with babylon (I'm not sure the `leadingComments` node is already initialized) and is the only point in the code we mutate a received parameter. If it's ok for you I'd re-enable the rule and I'll use an `eslint-disable-line` instruction. If you don't feel comfortable with this approach let the rule as it is now.

**Comment by MicheleBertoli** [comment]
Done @cef62, please merge it if it's ok now.
Thanks!

**Pull request merged**